core/classes/Core/Validate.php

```diff
@@ -318,6 +324,50 @@ public static function check(array $source, array $items = []): Validate {
                             break;
                         }
                         break;
+
+                    case self::RATE_LIMIT:
+                        if (is_array($rule_value) && count($rule_value) === 2) {
+                            // If array treat as [limit, seconds]
+                            [$limit, $seconds] = $rule_value;
+                        } else if (is_int($rule_value)) {
+                            // If integer default seconds to 60
+                            [$limit, $seconds] = [$rule_value, 60];
+                        }
+
+                        if (!isset($limit) || !isset($seconds)) {
+                            throw new Exception('Invalid rate limit configuration');
+                        }
+
+                        $session = $_SESSION["rate_limit_{$item}"];
```

Comment:
We are repeating the "rate_limit_{$item}" a few times here, can we create a variable like $rate_limit_key to not need to repeat it?